spec/server_spec.lua

```diff
@@ -50,16 +63,58 @@ describe("http.server module", function()
 		assert.truthy(cq:empty())
 		assert.spy(on_stream).was.called()
 	end
-	it("works with plain http 1.1", function()
+	it("works with plain http 1.1 using IP", function()
 		simple_test(false, 1.1)
 	end)
-	it("works with https 1.1", function()
+	it("works with https 1.1 using IP", function()
 		simple_test(true, 1.1)
 	end)
-	it("works with plain http 2.0", function()
+	it("works with plain http 2.0 using IP", function()
 		simple_test(false, 2.0)
 	end);
-	(require "http.tls".has_alpn and it or pending)("works with https 2.0", function()
+	(require "http.tls".has_alpn and it or pending)("works with https 2.0 using IP", function()
 		simple_test(true, 2.0)
 	end)
+	--[[
+	--
+	-- Until there is a way to generate OpenSSL contexts in this file for
+	-- UNIX domain sockets, there is no way to use TLS with this. Because
+	-- of this, the status for using TLS with UNIX domain sockets is
+	-- pending.
+	--
+	--]]
+	local socket_path
```

Comment:
I'm double checking now but Linux systems create the return value of os.tmpname() so I need to remove that, then use the filename with `.socket`. I'm checking later when I get home whether I actually use the path generated in the tests. I might have to create a wrapper function for this and then remove this block.